tasks/pkg-debian.yml

```diff
@@ -56,9 +56,6 @@
     state: "absent"
     update_cache: yes
   with_items:
-    - "deb http://apt.datadoghq.com/ stable main"
```

Comment:
These three should actually stay, as this task is supposed to ensure that non-https source is *absent*, not present.

Reply:
:facepalm: i should have spotted that. Patch updated.